Services/UI/OJS.Services.Ui.Business/Implementations/ContestsBusinessService.cs

```diff
@@ -144,6 +145,71 @@ public async Task<ContestDetailsServiceModel> GetContestDetails(int id)
             return contestDetailsServiceModel;
         }
 
+        public async Task<ContestLegacyExportServiceModel> Export(int id)
+        {
+            var contest = await this.contestsData
```

Comment:
Wouldn't it be more efficient to use `.AsSplitQuery()`, since we are joining a lot of tables.